app/templates/_README.md

```diff
@@ -0,0 +1,36 @@
+# <%= humanName %>
+
+<%= props.description %>
+
+![Screenshot](screenshot.png)
+
+## Dependencies
+
+- [Grunt](http://gruntjs.com): task management for pulling in assets,
```

Comment:
Let's change this to "task runner" to fit the format of the other three descriptions in this section.